tests/Jellyfin.Networking.Tests/NetworkParseTests.cs

```diff
@@ -61,6 +61,39 @@ public void IgnoreVirtualInterfaces(string interfaces, string lan, string value)
             Assert.Equal(value, "[" + string.Join(",", nm.GetInternalBindAddresses().Select(x => x.Address + "/" + x.Subnet.PrefixLength)) + "]");
         }
 
+        /// <summary>
+        /// Checks valid IP address formats.
+        /// </summary>
+        /// <param name="address">IP Address.</param>
+        [Theory]
+        [InlineData("127.0.0.1")]
+        [InlineData("fd23:184f:2029:0:3139:7386:67d7:d517")]
+        [InlineData("[fd23:184f:2029:0:3139:7386:67d7:d517]")]
+        [InlineData("fe80::7add:12ff:febb:c67b%16")]
+        [InlineData("[fe80::7add:12ff:febb:c67b%16]:123")]
+        [InlineData("fe80::7add:12ff:febb:c67b%16:123")]
+        [InlineData("[fe80::7add:12ff:febb:c67b%16]")]
+        [InlineData("192.168.1.2")]
```

Comment:
We might want include a [ULA](https://en.wikipedia.org/wiki/Unique_local_address) and perhaps also a  GUA for those that have native IPv6 connectivity ?

Reply:
Will check it tomorrow how those are handled (I think the code does handle them to some extent).

Reply:
Those are just parsing tests.
The code currently falls back to using ULA, Link Local and Loopback as local subnets if none are explicitly specified:
https://github.com/jellyfin/jellyfin/blob/dd5f90802e71083347b6095eb79a9de0b9d34615/Jellyfin.Networking/Manager/NetworkManager.cs#L306-L311

None of our UDP-based auto-detection and DLNA functionality currently works on IPv6.